runtime/lua/vim/lsp/builtin_callbacks.lua

```diff
@@ -214,14 +214,16 @@ BuiltinCallbacks['textDocument/hover'] = {
         end
       -- string
       else
-        table.insert(contents, data.contents)
+        for _, line in pairs(vim.api.nvim_call_function('split', { data.contents, '\\n' })) do
+          table.insert(contents, line)
```

Comment:
could use `:help vim.split()`. 

I also plan to make `tbl_extend()` work with list-like tables.